is_core/templates/base.html

```diff
@@ -10,6 +10,11 @@
 		{% block body %}
 			{% block sticky-flash %}
 				<div id="sticky-flash" class="flash">
+					{% block user-takeover %}
```

Comment:
This flash message will pop on every page that takeover user will look at? Wouldn't it be annoying?